frontends/ol-components/src/components/SortableList/SortableList.tsx

```diff
@@ -69,16 +69,17 @@ const SortableItem = <I extends UniqueIdentifier = UniqueIdentifier>(
   const handleProps: HandleProps = useMemo(
     () => ({
       ...listeners,
+      ...attributes,
       ref: setActivatorNodeRef,
       className: !props.disabled ? "ol-draggable" : undefined,
     }),
-    [setActivatorNodeRef, listeners, props.disabled],
+    [setActivatorNodeRef, attributes, listeners, props.disabled],
   )
 
   const { Component = "div" } = props
 
   return (
-    <Component ref={setNodeRef} style={style} {...attributes}>
+    <Component ref={setNodeRef} style={style}>
       <DragStyles>{props.children && props.children(handleProps)}</DragStyles>
```

Comment:
For some context: The reason there's a render function being uses here is that our sorting implementation supports   cases where the entire item is not draggable, but some "handle" is draggable. So the render function is used so that you can pass the listeners and attributes to the handle.

In practice, all the sort items we currently have are "whole item draggable". The handle behavior is somewhat legacy, but since a drag handle supports our current use case (whole item IS the handle) I don't see any reason to remove it.

**The bug:** The listeners and attributes need to be attached to the same item. Before we were doing something more akin to
```tsx
<div {...dragAttributes }>
  <div {...dragListeners />
</div>
```
The inner div had all the correct listeners (including keyboard listeners)!. But it had no tab index. So keyboard users had no way of actually interacting with the inner draggable div. The drag attributes and drag listeners need to be on the same div.

**Testing:** Unfortunately, I don't know of any reasonable way to test this on CI without a real browser :/